doc/source/user_guide_scripting/helper_scripts.rst

```diff
@@ -0,0 +1,342 @@
+- :ansyshelp:`Add the mesh refinement <act_script_demo_trans_struct.html>`
+- :ansyshelp:`Add face meshing to a named selection <act_script_demo_Linear_Periodic.html>`
+- :ansyshelp:`Add match control <act_script_demo_Cyclic.html>`
+
+------------------------
+
+- :ansyshelp:`Add a sweep method <act_script_demo_steady_state_therm.html>`
+- :ansyshelp:`Add a hex dominant method <act_script_demo_coupled_field_001.html>`
+- :ansyshelp:`Add a quad dominant method <act_script_demo_transient_therm.html>`
+- :ansyshelp:`Select nodes by IDs <act_script_examples_select_geom_or_mesh.html>`
+- :ansyshelp:`Given a node ID, get the node's information <act_script_examples_query_mesh.html>`
+- :ansyshelp:`Create a selection based on the location of nodes in Y <act_script_examples_create_selection_based_on_Location_of_nodes_in_Y.html>`
+- :ansyshelp:`Add a node merge group <act_script_demo_harmonic_acoustic.html>`
+- :ansyshelp:`Create a node merge object at a symmetry plane <act_script_examples_create_node_merage_object.html>`
+
+------------------------
+
+- :ansyshelp:`Suppress or un-suppress meshing objects <act_script_demo_Cyclic.html>`
+- :ansyshelp:`Mesh a model multiple times and track its metrics <act_script_examples_remesh_model.html>`
+- :ansyshelp:`Clear the mesh <act_script_examples_clear_mesh.html>`
+
+Loads and boundary conditions
+-----------------------------
+
+- :ansyshelp:`Create a pressure load <act_script_examples_create_pressure_load.html>`
+- :ansyshelp:`Use a named selection for a pressure <act_script_demo_Symmetric.html>`
+- :ansyshelp:`Apply spatially varying pressure on a named selection <act_script_demo_varying_load.html>`
+- :ansyshelp:`Change the tabular data values of loading condition <act_script_examples_change_tabular_data.html>`
+- :ansyshelp:`Use a named selection as scoping of a load or support <act_script_examples_NamedSelection_as_Scoping.html>`
+- :ansyshelp:`Add a remote force <act_script_demo_static_struct.html>`
+- :ansyshelp:`Add a thermal condition <act_script_demo_static_struct.html>`
+- :ansyshelp:`Add imported body temperature <act_script_demo_Cyclic.html>`
+
+------------------------
+
+- :ansyshelp:`Add compression-only support <act_script_demo_trans_struct.html>`
+- :ansyshelp:`Use a named selection for a fixed support <act_script_demo_Symmetric.html>`
+- :ansyshelp:`Add a fixed or frictionless support <act_script_demo_static_struct.html>`
+- :ansyshelp:`Add displacement BC <act_script_demo_trans_struct.html>`
+- :ansyshelp:`Add displacement BC 2 <act_script_demo_coupled_field_harmonic.html>`
+- :ansyshelp:`Add remote displacement <act_script_demo_coupled_field_001.html>`
+
+------------------------
+
+- :ansyshelp:`Add a bearing load <act_script_demo_trans_struct.html>`
+- :ansyshelp:`Add Earth gravity <act_script_demo_rbd_contact.html>`
+- :ansyshelp:`Add a command snippet <act_script_demo_coupled_field_transient.html>`
+- :ansyshelp:`Add PSD Acceleration  <act_script_demo_random_vib.html>`
```

Comment:
@klmcadams I've seen a few strays from sentence case but am assuming you are matching the case of the actual title in the help.